### PR TITLE
Quick Add WorkItem At Bottom

### PR DESCRIPTION
**Pull request by SMahil**
READY TO MERGE

### CODE REVIEW

File: src/app/work-item/work-item-quick-add/work-item-quick-add.component.ts [hunk 1]
```diff
@@ -36,20 +38,58 @@ export class WorkItemQuickAddComponent implements OnInit {
     } as WorkItem;
   }
 
-  save(): void {
+  //save(): void {
```
Comment (joshuawilson) [inline src/app/work-item/work-item-quick-add/work-item-quick-add.component.ts]: I think you need the code that you commented out here. That at least allows it to save and the enter button saves too.

File: src/app/work-item/work-item-list/work-item-list.component.html [hunk 2]
```diff
@@ -1,9 +1,10 @@
+<div class="workItem_Overlay" [ngClass]="{'activeOverlay' : showOverlay, 'inactivOverlay' : !showOverlay}"></div>
 <alm-app-header></alm-app-header>
 <div class="container-fluid screen-min-ht containerPaddT">
 	<div id="workItemList_quickAdd" class="width100 fl">
-		<div class="width100 fl supress-hover" (click)="addWorkItem()">
+		<!--<div class="width100 fl supress-hover" (click)="addWorkItem()">
```
Comment (joshuawilson) [inline src/app/work-item/work-item-list/work-item-list.component.html]: Unless you plan on going back to this design, you can delete these lines.

File: src/app/footer/footer.component.html [hunk 3]
```diff
@@ -1 +1 @@
-<div id="footer_container" class="footer width100 fl">Copyright 2016 Red Hat</div>
+<div id="footer_container" class="footer width100 fl dn">Copyright 2016 Red Hat</div>
```
Comment (joshuawilson) [inline src/app/footer/footer.component.html]: Why are you hiding the footer?

File: src/app/work-item/work-item-quick-add/work-item-quick-add.component.html [hunk 4]
```diff
@@ -1,5 +1,5 @@
-<div class="width100 fl marginT20">
-  <div id="workItemQuickAdd_container" class="width100 fl">
+<div class="width100 fl quickAdd_container" id="workItemQuickAdd_container" >
+  <!--<div id="workItemQuickAdd_container" class="width100 fl">
```
Comment (joshuawilson) [inline src/app/work-item/work-item-quick-add/work-item-quick-add.component.html]: There is no need to keep a commented version if you don't plan on going back to it. Git history is usually enough to find the way it worked in the past.

File: src/app/work-item/work-item-list/work-item-list.component.ts [hunk 5]
```diff
@@ -80,4 +81,10 @@ export class WorkItemListComponent implements OnInit {
   onDelete(entryComponent: WorkItemListEntryComponent): void {
     this.reloadWorkItems();
   }
+
+  toggleOverlay(val:boolean) : void{
```
Comment (joshuawilson) [inline src/app/work-item/work-item-list/work-item-list.component.ts]: TSLint says you need a space in this line.

File: src/app/work-item/work-item-quick-add/work-item-quick-add.component.ts [hunk 6]
```diff
@@ -34,15 +39,19 @@ export class WorkItemQuickAddComponent implements OnInit {
...
     } as WorkItem;
+    this.showQuickAdd = false;
+    this.showQuickAddBtn = true;
   }
 
   save(): void {
     if (this.workItem.fields['system.title'] != null) {
       this.workItem.fields['system.title'] = this.workItem.fields['system.title'].trim();
-    }
+   }
```
Comment (joshuawilson) [inline src/app/work-item/work-item-quick-add/work-item-quick-add.component.ts]: For consistency, this bracket should be moved back over one space.

**Comment by joshuawilson** [comment]
@SMahil in the PR that I sent you last week there was an "X" for closing the quick-add. I don't see that anymore. The design from UX had that in there. Currently the design does not match the desired work flow. Currently it is: click button, click text box, type text, hit enter (to enter more you need to start over by clicking on the button). The work flow should have something more like click button, type text, hit enter, type text, hit enter, type text, hit enter, click close icon when done.
https://redhat.invisionapp.com/share/798MZ3GG5#/screens/185961157

File: src/app/footer/footer.component.html (hunk 3 above)
Comment (SMahil) [inline src/app/footer/footer.component.html]: There is no design or workflow for footer when we have Quick Add At Bottom, So I just hide it

**Comment by nimishamukherjee** [comment]
[test]

**Pull request merged**